_posts/2021-11-08-hhvm-4.136.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: "HHVM 4.136"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.136 is released! HHVM 4.131&ndash;4.135 remain supported, as do the 4.102 and 4.128 LTS releases.
+
+# Highlights
+
+- Add quickfix for missing interface methods.
+- [The `readonly` feature](https://docs.hhvm.com/hack/experimental-features/read-only) is no longer considered unstable.
```

Comment:
https://docs.hhvm.com/hack/readonly/introduction instead; we should delete/redirect the old page (cc @jamesjwu)